Library/Homebrew/formula_installer.rb

```diff
@@ -212,6 +212,11 @@ def prelude
   def verify_deps_exist
     begin
       compute_dependencies
+    rescue CoreTapFormulaUnavailableError => e
+      raise unless Homebrew::API::Bottle.available? e.name
```

Comment:
Should this be scoped to `Homebrew::EnvConfig.install_from_api?`?

Reply:
No need because `CoreTapFormulaUnavailableError` is only raised if `Homebrew::EnvConfig.install_from_api?`. Otherwise, it's a standard `TapFormulaUnavailableError`.

Another reason I don't love this setup is that this isn't very clear which makes it hard to figure out what's going on by looking at the code. I know that this is what happens because I wrote it, but it's not surprising that you'd have this confusion.

Reply:
> No need because `CoreTapFormulaUnavailableError` is only raised if `Homebrew::EnvConfig.install_from_api?`. Otherwise, it's a standard `TapFormulaUnavailableError`.

That's the case now but I'm not sure it's worth relying on that consistently.



> Another reason I don't love this setup is that this isn't very clear which makes it hard to figure out what's going on by looking at the code. I know that this is what happens because I wrote it, but it's not surprising that you'd have this confusion.

Agreed 👍🏻. Fine for now.